cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -386,6 +388,33 @@ function DownloadAndInstall-KubernetesBinaries {
   Remove-Item -Force -Recurse $tmp_dir
 }
 
+# Downloads the csi-proxy binaries from kube-env's CSI_PROXY_STORAGE_PATH and
+# CSI_PROXY_VERSION, and then puts them in a subdirectory of $env:NODE_DIR. 
+# Note: fow now the installation is skipped for non-test clusters. Will be
+# installed for all cluster after tests pass.
+# Required ${kube_env} keys:
+#   CSI_PROXY_STORAGE_PATH and CSI_PROXY_VERSION
+function DownloadAndInstall-CSIProxyBinaries {
+  #if (Test-IsTestCluster $kube_env) {
+    if (ShouldWrite-File ${env:NODE_DIR}\csi-proxy.exe) {
+      $tmp_dir = 'C:\k8s_tmp'
+      New-Item -Force -ItemType 'directory' $tmp_dir | Out-Null
+      $filename = 'csi-proxy.exe'
+      $urls = "${env:CSI_PROXY_STORAGE_PATH}/${env:CSI_PROXY_VERSION}/$filename"
```

Comment:
This will work, but it will be annoying if/when the path structure changes in the future. Should we just write the full url in cluster/gce/config-common.sh and pass it through? Is there a reason not to do that?

Reply:
I just followed the structure of WINDOWS_CNI_STORAGE_PATH and WINDOWS_CNI_VERSION. Later when we need to pass version information from GKE, I am wondering whether this makes difference?

Reply:
I guess it's fine either way. Not sure why we did it that way for CNI.